config/initializers/prometheus_exporter.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-if Rails.env.production?
+if ENV['PROMETHEUS_METRICS'].present?
```

Comment:
One minor thing after looking at https://github.com/ministryofjustice/hmpps-book-secure-move-api-deploy/pull/38#pullrequestreview-361761369. We're checking for presence, but using `on`. Would it be worth checking for `"on"`, `"true"`, or `true` here?